.firebase/hosting.ZGlzdFxlYXQtaXQtYWxs.cache

```diff
@@ -0,0 +1,8 @@
+index.html,1608014145387,ae185f077cda611f3993139d5f608257d3347516d5ae0c17c837df1c7141d9ff
+3rdpartylicenses.txt,1608014144745,d0cc094c6b35fca71fe4531c69b8b598dd8860cbb40b80e022993e65d80576bc
+runtime.67a341bf59ff7a6c70c2.js,1608014144745,6caf2a42a433739dd9b89bf52fa9bb64af6a36d702ce54667a7713ce53912b5e
+polyfills.37ca02d7b75d37413343.js,1608014144745,a434fd32b8a9642ed400267220ca403a633cd49f1388d8c08d3890a3f6e66492
+favicon.ico,1607006839025,2c19690e9587bae12f419b34d2edeecc76808099540a9c9f4ea6194116cfc8f7
+styles.ef572175d5b280ca104a.css,1608014144745,e1d74d8033336ff3eba5161582b12121d4546efc5a6956e24dd4a7eb4c3ce4c4
+4.4e1cd510a555abbfb48c.js,1608014144745,90f721c15ddbbb27edec6f035f9a5a088ed922c6d705298426d6e466f7afaa58
+main.e8e76616ff5fb185aa43.js,1608014144745,984064280a9b7e6b7c7db93341d57ec87f42291b8275a62cfd07f165387a154c
```

Comment:
This probably doesn't need to be commited.

Reply:
Oh yeah. I missed that